src/components/TopicLoader/TopicLoader.js

```diff
@@ -105,13 +105,6 @@ class TopicLoader extends Component {
       if (apiKey && flatLayers[i] instanceof TrafimageRasterLayer) {
         flatLayers[i].setApiKey(apiKey);
       }
-
-      if (
-        flatLayers[i] instanceof VectorLayer ||
-        flatLayers[i] instanceof WMSLayer
-      ) {
-        flatLayers[i].onClick(this.onClick.bind(this));
```

Comment:
it seems we add  the same callbacks over and over without removing the old ones.

Reply:
pff forget these lines are removed not added :D